### ISSUE TITLE
redeclared in this block

### ISSUE THREAD

**Issue by riramar** [OWASP/Amass#265]
```
root@zion:~# go version
go version go1.13 linux/amd64
root@zion:~# go get -u github.com/OWASP/Amass/...
# runtime
/usr/local/go/src/runtime/stubs_x86.go:10:6: stackcheck redeclared in this block
	previous declaration at /usr/local/go/src/runtime/stubs_amd64x.go:10:6
/usr/local/go/src/runtime/unaligned1.go:11:6: readUnaligned32 redeclared in this block
	previous declaration at /usr/local/go/src/runtime/alg.go:321:40
/usr/local/go/src/runtime/unaligned1.go:15:6: readUnaligned64 redeclared in this block
	previous declaration at /usr/local/go/src/runtime/alg.go:329:40
root@zion:~# amass -version
v3.0.25
root@zion:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Kali
Description:	Kali GNU/Linux Rolling
Release:	2019.4
Codename:	kali-rolling
root@zion:~# 
```

**Comment by caffix** [comment]
Your build environment seems to have some problems (Go runtime source code). Also, please update your Amass source to v3.1.5

**Comment by riramar** [comment]
Thanks! For issue #264 just deleting the Amass src solved the problem. I've never deleted the Amass src before to update the version.
I've checked here https://golang.org/cmd/go/ under "go get" if there is any flag to refresh the local repo but didn't find anything. Is there a right go command to update the version in this case without manually deleting the src?

For this issue just deleting the Amass src didn't work. I've solved this issue following the instructions from this post https://golangnote.com/topic/263.html. It seems is the only reference that Google found about this problem.
